portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,6 +49,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     for(Entity entity : results.asIterable()) {
       String commentContent = (String) entity.getProperty(CONTENT_TEXT_PROPERTY_NAME);
       demoComments.add(commentContent);
+
+      if(demoComments.size() == numCommentsDisplayed)  {
```

Comment:
It is a good practice to validate the parameters at server side. e.g. I can manually change the param to -1 in the url and the server now pulls all comments for me, even though your intention is to give me 10 comments at most.

Reply:
I see, I'll fix this in the next PR, thank you!